src/components/ui/MantineTable.tsx

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from 'react';
+import {
+  MantineReactTable,
+  useMantineReactTable,
+  type MRT_ColumnDef, //if using TypeScript (optional, but recommended)
+} from 'mantine-react-table';
+
+//If using TypeScript, define the shape of your data (optional, but recommended)
+interface Compare {
+  optionA: string;
+  delta: string;
+  optionB: string;
+}
+
+//mock data - strongly typed if you are using TypeScript (optional, but recommended)
+const data: Compare[] = [
+  {
+    optionA: '69',
+    delta: 'B is better',
+    optionB: '420'
+  },
+  {
+    optionA: '69',
+    delta: 'B is better',
+    optionB: '420'
+  },
+];
+
+export default function App() {
```

Comment:
Let's stick to our usual convention of declaring Functional Components, and also please name the component properly :)
```suggestion
export const PlaceComparisonTable: React.FC = () => {
```